### PR TITLE
Add Mode sUSDe to Address Book and modesUSDe price oracleId

### PR DESCRIPTION
**Pull request by iamjackgale**

### CODE REVIEW

File: src/utils/fetchConcentratedLiquidityTokenPrices.ts [hunk 1]
```diff
@@ -491,6 +491,14 @@ const tokens: Partial<Record<keyof typeof ChainId, ConcentratedLiquidityToken[]>
       firstToken: 'XVELO',
       secondToken: 'WETH',
     },
+    {
+      type: 'Slipstream',
+      oracleId: 'modesUSDe',
+      decimalDelta: 1,
+      pool: '0xa2E74518fe27B0FC04Ea36024ea837d26cd09CDa',
+      firstToken: 'sUSDe',
```
Comment (kexleyBeefy) [inline src/utils/fetchConcentratedLiquidityTokenPrices.ts:499]: Switch the order and `sUSDe` should be `modesUSDe`

File: src/utils/fetchConcentratedLiquidityTokenPrices.ts [hunk 2]
```diff
@@ -491,6 +491,14 @@ const tokens: Partial<Record<keyof typeof ChainId, ConcentratedLiquidityToken[]>
       firstToken: 'XVELO',
       secondToken: 'WETH',
     },
+    {
+      type: 'Slipstream',
+      oracleId: 'modesUSDe',
+      decimalDelta: 1,
```
Comment (kexleyBeefy) [inline src/utils/fetchConcentratedLiquidityTokenPrices.ts:497]: delta is `1e12`

File: packages/address-book/src/address-book/mode/tokens/tokens.ts [hunk 3]
```diff
@@ -142,4 +142,19 @@ export const tokens = {
     documentation: 'https://velodrome.finance/docs',
     bridge: 'superchain',
   },
+  sUSDe: {
+    name: 'Staked USDe',
+    symbol: 'sUSDe',
+    oracleId: 'sUSDe',
```
Comment (kexleyBeefy) [inline packages/address-book/src/address-book/mode/tokens/tokens.ts:148]: oracle id should be `modesUSDe`

**Review by kexleyBeefy — CHANGES_REQUESTED** [review]

**Review by iamjackgale — COMMENTED** [review]
All changes implemented in latest commit

**Pull request merged**